### ISSUE TITLE
Monitor mapping broken on Gnome 41.1

### ISSUE THREAD

**Issue by X9VoiD** [OpenTabletDriver/OpenTabletDriver#1660]
## Description

OTD monitor mapping is broken on Gnome 41.1 and is constrained to a single display.

**Comment by InfinityGhost** [comment]
Is this Wayland or Xorg? I'm unable to test this as I've only got a single display setup.

**Comment by X9VoiD** [comment]
Both.

**Comment by gonX** [comment]
Possibly duplicate of #1137?

**Comment by gonX** [comment]
According to the reporter in #1137 this seems to no longer be the case. Can you confirm @X9VoiD ?

**Comment by gonX** [comment]
I've tested this on GNOME 41.2 and I am unable to reproduce

**Comment by gonX** [comment]
Reopening since someone on 41.2 seems to have the issue

[discord convo link](https://discord.com/channels/615607687467761684/723399565780189234/924617534949441556)

**Comment by gonX** [comment]
Potential leads: artist mode and left (mapped) monitor mapped below right monitor

**Comment by sburris0** [comment]
I believe I'm experiencing this on GNOME 44.2/Wayland/nixos.

From what I can tell, this bug is centered around configured display positions. I don't notice a difference between absolute and artist mode with my Huion Kamvas 16 2021. Pen position looks correct in tablet debugger view.

I've played with monitor positions a lot, and the cursor seems to be:
* too high or too low, depending on monitor positions
* always too far to the left, varying amounts

Note that tablet is monitor 2 in the below images

## Cases of incorrect position
Cursor too high and to the left:
![Screenshot from 2023-07-06 21-43-34](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/0b948c32-ed62-4751-81c0-f646632cef28)

Cursor WAY too high and to the left:
![Screenshot from 2023-07-06 21-44-18](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/bca8523e-e3fd-4e60-8075-6202a4422e48)

Cursor too low and to the left:
![Screenshot from 2023-07-06 21-45-06](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/acddfdaa-6530-4168-83ad-f3254fd57870)

Cursor too low and WAY too far to the left:
![Screenshot from 2023-07-06 21-45-56](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/a0988f09-ba12-48b7-b177-8d6de13549fb)

Cursor too high and to the left:
![Screenshot from 2023-07-06 21-46-56](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/4a604ea8-b484-44cb-ae1e-ea406ac421a0)

Cursor very high and way too far to the left:
![Screenshot from 2023-07-06 21-48-09](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/2de7febd-2113-4ec2-969d-4da50128e6cd)

## Cases of correct position
If I re-enable my third display, I can actually get the correct cursor positions with these configurations:
![Screenshot from 2023-07-06 21-56-49](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/f1a89e0b-e049-442b-9724-8dc5592c48ff)
![Screenshot from 2023-07-06 21-56-20](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/c9a336c7-a371-49b8-b402-8adbe7b348a4)
![Screenshot from 2023-07-06 21-58-20](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/a11766fb-d11a-432e-a6ff-965e2722183f)
![Screenshot from 2023-07-06 21-57-53](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/58098832/4bc1e87c-f151-4e9e-998b-9a675b83ce61)

But no others that I have found.

This seems pretty reliable on my setup, so I'd be happy to run experiments or provide any other info

**Comment by jamesbt365** [comment]
Not really seeing any complaints about mapping not working on gnome anymore? I assume this is probably fixed upstream already?

**Comment by gonX** [comment]
> Not really seeing any complaints about mapping not working on gnome anymore? I assume this is probably fixed upstream already?

Yeah, going to assume the same. I've not heard many GNOME complaints the last couple of months.

Closing as assumed fixed upstream.
